preparing-for-ada/applying-to-ada/readme.md

```diff
@@ -1 +1,3 @@
-Coming soon ...
\ No newline at end of file
+Applicants who have completed this JumpStart curriculum including all the coding exercises, are better prepared for the application process as well as to start as a new student at Ada Developers Academy.
```

Comment:
maybe change the second part to "...application process and therefore becoming a new student at Ada Developers Academy."

Reply:
Makes sense. Done!